pinctrl/pinctrl_utils.go

```diff
@@ -246,16 +265,16 @@ func (b *pinctrlpi5) setGPIONodePhysAddr(nodePath string) error {
 	/* Call recursive function to calculate phys addr. Works way up the device tree, using the information
 	found in #ranges at every node to translate from the child's address space to the parent's address space
 	until we get the child's physical address in all of /dev/gpiomem. */
```

Comment:
this should say all of /dev/mem

Reply:
should all instances of /dev/gpiomem be renamed to /dev/mem? including the name of the variable in the struct?